594 Minor content changes

jema28 opened the pull request:
#### Issues this PR relates to

#594

jema28 commented:
@thejoefriel I've had to reduce the body text to 14px to make it fit as it was still too long.

jema28 commented:
Will sort the rest of the changes now.

jema28 commented:
@thejoefriel I've made the other content changes, now ready for review.

thejoefriel commented on client/src/pages/GiveCredits/earnActivityCategories.js:
```diff
@@ -2,23 +2,23 @@
     subtitle: "e.g. working on reception or completing paperwork",
   },
   {
-    category: "developingServices",
```
Why have all the categories been changed as well? This means that the icons are no longer working for the ones you've changed: http://www.clipular.com/posts/4892976940318720?k=WytxaXpXqwqZhDyWTSXcsaMVn_s

Please change that back unless there's a reason?

thejoefriel commented:
> @thejoefriel I had to reduce the body text to 14px to make it fit as it was still too long.

OK cool. @harrietepugh due to there being more text in the welcome screens we've had to bring the font size down slightly. Are you ok with this? https://www.loom.com/share/65100693477b4f8b8cf9de21f9c5d4a0

harrietepugh commented:
@thejoefriel @jema28 can we make the images smaller? Reluctant to make font smaller as this came up in testing! Lots of older people found it small as it is!

jema28 commented:
Hi @thejoefriel, I've made those changes, ready for review:
- reverted the categories back to the originals
- changed earn group nav link and reverted headline to original (changed it by mistake)
- reduced the size of the image to 230px instead of 250px
